routes/web.php

```diff
@@ -45,6 +46,19 @@
     });
 });
 
+
+Route::prefix('/lesson')->group(function () {
+    Route::get('/', [LessonController::class, 'index'])->name('lesson.index');
+    Route::get('/show/{id}', [LessonController::class, 'show'])->name('lesson.show');
+    Route::get('/signup/{id}', [RegistrationController::class, 'userSignUp'])->name('signups.public.signup');
+    Route::post('/dosignup/{lesson_id}/{user_id}', [RegistrationController::class, 'doUserSignup'])->name('signups.public.doSignup');
+    Route::get('/registrations', [RegistrationController::class, 'userIndex'])->name('signups.public.index');
```

Comment:
Once again, why is the registration page even available to the public and not just members? you may have a reason idk :)

Reply:
Fixed